test/libsolidity/lsp/include-paths/bad_include.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+   import "test.sol";
+// ^^^^^^^^^^^^^^^^^^ @BadInclude
```

Comment:
As discussed on the chat, the include is valid, it's just that the file is missing - so the error (and test name) should say just that.

Reply:
Also, we should have a positive test for this case too. I.e. a file placed directly under base path or one of the include paths should be successfully imported.